front-end-master/landing_page/index.css

```diff
@@ -409,7 +409,7 @@ a, a:hover{
 
 /*------------------------------*/
 .contact-card-container{
-	max-width: 710px !important;
+	max-width: 700px !important;
```

Comment:
I don't like the use of !important. Is this absolutely necessary?

Reply:
yes, I'm overriding the original class.